packages/main/src/plugin/install/extension-installer.ts

```diff
@@ -120,16 +126,28 @@ export class ExtensionInstaller {
       return;
     }
 
-    const titleLabel = imageConfigLabels['org.opencontainers.image.title'];
+    const titleLabel = imageConfigLabels['org.opencontainers.image.title'] as string | undefined;
     const descriptionLabel = imageConfigLabels['org.opencontainers.image.description'];
     const vendorLabel = imageConfigLabels['org.opencontainers.image.vendor'];
     const apiVersion = imageConfigLabels['io.podman-desktop.api.version'];
+    const apiDDVersion = imageConfigLabels['com.docker.desktop.extension.api.version'];
 
-    if (!titleLabel || !descriptionLabel || !vendorLabel || !apiVersion) {
+    if (!titleLabel || !descriptionLabel || !vendorLabel || (!apiVersion && !apiDDVersion)) {
       sendError(`Image ${imageName} is not a Podman Desktop Extension`);
```

Comment:
Message should be updated

Reply:
well it seems we wanted to abstract that everything is a Podman Desktop extension  (don't matter if it's podman or docker) so if message is updated it will bring back the issue no ?